sickbeard/providers/newznab.py

```diff
@@ -141,9 +141,11 @@ def _get_season_search_strings(self, ep_obj):
 
         # search
         rid = helpers.mapIndexersToShow(ep_obj.show)[2]
+        #added exception for usenet-crawler, as it seems they have changed something within their api
+        tvrage = 'rid' if self.name <> "Usenet-Crawler" else 'tvrage' 
```

Comment:
To be pyhtoninc you should only use != (cstyle comparison operators) when comparing integers

I would rather see 
```python
tvrage = 'rid' if self.name not in "Usenet-Crawler" else 'tvrage' 
```
or
```python
tvrage = ('rid', 'tvrage') ["Usenet-Crawler" in self.name] 
```